komga-webui/src/locales/en.json

```diff
@@ -232,37 +231,37 @@
     "book_number": "Book number: {name}",
     "book_series": "Series: {name}",
     "button_import": "Import",
-    "comicrack_preambule_html": "You can import existing ComicRack Reading Lists in <code>.cbl</code> format.<br/>Komga will try to match the provided series and book number with series and books in your libraries.",
+    "comicrack_preambule_html": "When import existing ComicRack Reading Lists in <code>.cbl</code> format<br/>Komga tries to match the provided series and book number with series and books in your libraries.",
     "field_files_label": "ComicRack Reading Lists (.cbl)",
     "import_read_lists": "Import Read Lists",
     "imported_as": "Imported as {name}",
-    "results_preambule": "Result of the import is shown below. You can also check the unmatched books for each provided file.",
-    "size_limit": "Size should be less than {size} MB",
+    "results_preambule": "Result of the import is shown below. You can check the unmatched books for each provided file.",
+    "size_limit": "The size should be less than {size} MB",
     "tab_title": "Data Import"
   },
   "dialog": {
     "add_to_collection": {
       "button_create": "Create",
-      "card_collection_subtitle": "No series | 1 series | {count} series",
+      "card_collection_subtitle": "No series | One series | {count} series",
       "dialog_title": "Add to collection",
       "field_search_create": "Search or create collection",
       "field_search_create_error": "A collection with this name already exists",
       "label_no_matching_collection": "No matching collection"
     },
     "add_to_readlist": {
       "button_create": "Create",
-      "card_readlist_subtitle": "No book | 1 book | {count} books",
+      "card_readlist_subtitle": "No book | One book | {count} books",
```

Comment:
For consistency i think it would be better to keep these as "1 page" and "1 pending task".

As once there is more then one it will be "4 pages" for example, so using both the actual number and the text representation of it in the same area would be a little odd.